### PR TITLE
add $GOPATH/bin to path and activate service account in runner

### PR DESCRIPTION
**Pull request by krzyzacy**
workaround for now.

ref https://github.com/kubernetes/test-infra/issues/9471, https://github.com/kubernetes/test-infra/issues/9469

/assign @BenTheElder @cjwagner

### CODE REVIEW

File: images/bootstrap/runner.sh [hunk 1]
```diff
@@ -91,6 +91,14 @@ fi
 
 # disable error exit so we can run post-command cleanup
 set +o errexit
+
+# add $GOPATH/bin to $PATH
+export PATH=$GOPATH/bin:$PATH
```
[the remark this answers is not in the document]
Comment (BenTheElder) [reply images/bootstrap/runner.sh]: https://github.com/kubernetes/test-infra/blob/2f64256fa26e76084c2f6b39e1575aa6e43fde18/prow/pod-utils/decorate/podspec.go#L46

just undocumented :-)

[the remark this answers is not in the document]
Comment (sebastienvas) [reply images/bootstrap/runner.sh]: Yes @BenTheElder podutils sets GOPATH.

**Comment by krzyzacy** [comment]
hey yo great...

https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/Dockerfile#L25-L27

I probably should hard code /home/prow/go/bin ?!?!

**Comment by krzyzacy** [comment]
thoughts? this-is-fine?

**Comment by krzyzacy** [comment]
/hold cancel
:crossed_fingers: (not going to bump oss yet)

**Pull request merged**